src/gc/gc.h

```diff
@@ -108,6 +108,7 @@ extern "C" uint8_t* g_gc_highest_address;
 extern "C" bool g_fFinalizerRunOnShutDown;
 extern "C" bool g_built_with_svr_gc;
 extern "C" uint8_t g_build_variant;
+extern "C" VOLATILE(int32_t) g_no_gc_lock;
```

Comment:
yep. gc.h should only contain interface related stuff, not implementation only stuff.

---
In reply to: [105787160](https://github.com/dotnet/coreclr/pull/10065#discussion_r105787160) [](ancestors = 105787160)